pkg/kubernetes/details.js

```diff
@@ -25,20 +28,84 @@ define([
 
         e.stopPropagation();
 
-        delete_btn.on('click', function() {
+        delete_entity_btn.off('click').on('click', function() {
             var promise = k8client.remove(entity.metadata.selfLink);
             delete_entity_dlg.dialog("promise", promise);
         });
     });
 
-    adjust_entity_dlg.on('show.bs.modal', function(e) {
+    adjust_route_dlg.on('show.bs.modal', function(e) {
         var key = $(e.relatedTarget).attr("data-key");
         var entity = k8client.objects[key];
-        adjust_entity_dlg.find('#replica-count').val(entity.spec.replicas);
+        adjust_route_dlg.find('#host-value').val(entity.spec.host);
```

Comment:
Any reason not to use these ids directly instead of using .find to select them? To make them less likely to conflict you could prefix them with the dialog id ei: adjust-route-dialog-host-value